decentraland/unity-renderer
# Feat: added favorite button in places card

## What does this PR change?

Fix #4079
Adds, behind a feature flag, the new favorite button in places card
...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/favorites-button&ENABLE_FAVOURITE_PLACES
2. Verify that a heart icon appears in all places cards and in the place info modal

1. Go to: https://play.decentraland.zone/?explorer-branch=feat/favorites-button
2. Verify that the heart icon doesn't appear in all places cards and in the place info modal

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
